app/templates/content/buyers-guide-g-cloud.html

```diff
@@ -0,0 +1,247 @@
+<p>Updated <abbr class="published-at date" title="2014-09-30T12:35:08+00:00">30 September 2014</abbr></p>
+</div>
+<nav class="in-page-navigation"><h2>Contents</h2>
+<ol>
+<li><a href="#buying-services-on-digital-marketplace">Buying cloud services on the Digital Marketplace</a></li>
+<li><a href="#g-cloud-order-form">Choosing the right G-Cloud framework agreement</a></li>
+<li><a href="#continuing-or-ending-g-cloud-services">Continuing or ending G-Cloud services</a></li>
+<li><a href="#after-your-contract-expires">After your contract expires</a></li>
+</ol></nav>
+</div>
+</header>
+<div class="block publication-content">
+<div class="inner-block floated-children">
+<div class="govspeak">
+
+<h2 id="buying-services-on-digital-marketplace">
+<span class="number">1. </span>Buying cloud services on the Digital Marketplace</h2>
+<p>
+  You can use the Digital Marketplace to find and compare cloud technology and support (eg web hosting or site analytics).
+</p><p>
+  All public sector organisations, including agencies and arm’s length bodies, can use the Digital Marketplace.
+  <a href='https://www.gov.uk/government/publications/public-sector-organisations-eligible-to-use-cloudstore'>Check the list of approved organisations</a> to see if you’re eligible.
+</p><p>
+  All the cloud services included on the Digital Marketplace are listed on the G-Cloud framework, and all suppliers have successfully been through the Official Journal of the European Union (OJEU) process. That means you can save on the time and cost traditionally associated with entering into individual procurement contracts.
+</p><p>
+  There are over 19,000 services on the Digital Marketplace, split into 4 categories:
+</p>
+<ul>
+  <li>
+    Infrastructure as a Service (IaaS)
+  </li><li>
+    Platform as a Service (PaaS)
+  </li><li>
+    Software as a Service (SaaS)
+  </li><li>
+    Specialist Cloud Services (SCS)
+  </li>
+</ul>
+<p>
+  When using the Digital Marketplace, you're still running your own procurement as usual and must still follow best practice.
+</p>
+<p>You should follow the buying process below:</p>
+
+<ol>
+<li>Prepare your requirements – a list of ‘must-haves’ and ‘wants’ – and get approval to buy what you need.</li>
+<li>Search for services on the Digital Marketplace using keywords.</li>
+<li>Review services and apply the filters to create your shortlist.</li>
+<li>Evaluate your options to find the cheapest or best value for money.</li>
+<li>Select your service, award and sign the call-off agreement.</li>
+<li>Complete the <a href="https://www.google.com/url?q=https%3A%2F%2Fccs.cabinetoffice.gov.uk%2Fsites%2Fdefault%2Ffiles%2Fcontracts%2FG-Cloud%2520Customer%2520Savings%2520Record_0.docx&sa=D&sntz=1&usg=AFQjCNGLkRRlQD5eJSozbjPRBNBaNuluIQ">Crown Commercial Service (CCS) savings form</a>.</li>
+</ol>
+
+<figure class="image embedded"><div class="img"><img alt="CloudStore buying process" src="/assets/compressed/buying-process.png"></div>
```

Comment:
This image isn't working.
![screen shot 2015-05-13 at 10 47 55](https://cloud.githubusercontent.com/assets/6525554/7607944/8caefef6-f95d-11e4-84f7-24347df2c053.png)